packages/amplify-e2e-tests/src/__tests__/analytics.test.ts

```diff
@@ -17,14 +17,23 @@ describe('amplify add analytics', () => {
     deleteProjectDir(projRoot);
   });
 
-  it('add pinpoint', async () => {
+  it('add pinpoint for javascript', async () => {
     await initJSProjectWithProfile(projRoot, {});
     const rightName = 'myapp';
     await addPinpoint(projRoot, { rightName, wrongName: '$' });
     await amplifyPushUpdate(projRoot);
     expect(fs.existsSync(path.join(projRoot, 'amplify', 'backend', 'analytics', rightName))).toBe(true);
   });
 
+  it('add pinpoint for flutter', async () => {
+    await initFlutterProjectWithProfile(projRoot, { name: 'storageTest' });
+    const rightName = 'myapp';
+    await addPinpoint(projRoot, { rightName, wrongName: '$' });
```

Comment:
No need to test `wrongName` here. Please remove.

Reply:
The test actually fails if I remove this.